packages/theme/include/icon.ts

```diff
@@ -0,0 +1,8 @@
+export const mediumHeight = {
+  height: 22
+}
+
+//icon images
```

Comment:
No comments in code. Really: http://www.codeodor.com/index.cfm/2008/6/18/Common-Excuses-Used-To-Comment-Code-and-What-To-Do-About-Them/2293